### ISSUE TITLE
sls variables are being replaced in v1.6.0

### ISSUE THREAD

**Issue by nicka** [svdgraaf/serverless-pseudo-parameters#20]
Having some issues after upgrading to v1.6.0 😢 

### serverless.yml

```yml
custom:
  config: ${file(config/${opt:stage}.js):config, file(config/common.js):config}

provider:
  ...
  environment:
    MEDIA_SERVICES_REGION: ${self:custom.config.mediaServices.region}
  ...
```

### config/common.js

```js
'use strict';

module.exports.config = () =>
  Object({ 
    mediaServices: { region: 'ap-southeast-1' }
  });
```

### Output

```js
FooLambdaFunction: {
       Properties: {
         ...
         Environment: {
           Variables: {
-            MEDIA_SERVICES_REGION: "ap-southeast-1"
+            MEDIA_SERVICES_REGION: {
+              Fn::Sub: "${AWS::Region}"
+            }
           }
         }
       }
```

In my case `${AWS::Region}` resulted in `ap-northeast-1` which caused applciation issues 😂

**Comment by svdgraaf** [comment]
I think your hit by the "auto replace all relevant region names" to the pseudo parameter one, you can set this flag to disable it:

```
custom:
    pseudoParameters:
        skipRegionReplace: true
```

That should fix your issue I think.

**Comment by svdgraaf** [comment]
You're not the first one to be hit by this, so perhaps it should be disabled by default in the next major release.

**Comment by svdgraaf** [comment]
I just released a new version of the plugin (`2.1.0`). This "feature" is now disabled by default, so you don't have to explicitly set it anymore.
